packages/hosted-form-v2/src/iframe-content/hosted-card-expiry-input.ts

```diff
@@ -25,7 +25,7 @@ export default class HostedCardExpiryInput extends HostedInput {
         eventPoster: IframeEventPoster<HostedInputEvent>,
         inputAggregator: HostedInputAggregator,
         inputValidator: HostedInputValidator,
-        paymentHandler: HostedInputPaymentHandler,
+        manualOrderPaymentHandler: HostedInputManualOrderPaymentHandler,
```

Comment:
Hi Richa, could you please add a comment documenting the decision and reasoning for using `manualOrderPaymentHandler` instead of `paymentHandler` here?

Reply:
Right now this package is used for manual order only, in future when we implement it for checkout-js we can refactor the code and name it `paymentHandler` if the logic to derive the handler based on some params is easier.